cypress/support/commands/form.js

```diff
@@ -11,40 +11,40 @@ function removeSubstringsWithCurlyBrackets(stringValue) {
 }
 
 Cypress.Commands.add('clearField', (fieldName, modal) => {
-    cy.log(`clearField - fieldName=${fieldName}; modal=${modal}`);
+  cy.log(`clearField - fieldName=${fieldName}; modal=${modal}`);
```

Comment:
I have the feeling i broke the indentation in this file, but only because eslint complained it didn't like 2 spaces, and wanted 4. 
No idea why that happened, sorry :(